pvl/decoder.py

```diff
@@ -193,8 +193,16 @@ def decode_datetime(self, value: str):
         seconds.  However, the Python ``datetime`` classes don't
         support second values for more than 59 seconds.
 
+        Since the PVL Blue Book says that all PVl Date/Time Values
+        are represented in Universal Coordinated Time, then all
+        datetime objects that are returned datetime Python objects
+        should be timezone "aware."  A datetime.date object is always
+        "naive" but any datetime.time or datetime.datetime objects
+        returned from this function will be "aware."
+
         If a time with 60 seconds is encountered, it will not be
-        returned as a datetime object, but simply as a string.
```

Comment:
I don't think it is worth addressing for the moment, but the behavior here was interesting to me as I have read things that indicate the time object does support 61 seconds https://docs.python.org/3/library/time.html#time.struct_time. But I haven't looked into it deeply to know if the string return type can be dropped (it would be cleaner if it can be)

Reply:
Yeah, that's part of a multi-para docstring for the base [decode_datetime()](https://pvl.readthedocs.io/en/latest/pvl.html#pvl.decoder.PVLDecoder.decode_datetime) method.  The very next paragraph talks about `time.struct_time` vs `datetime` objects.  There are subtleties.  Basically, if there is only a time (like "23:01:60") it would get parsed to a `time.struct_time` that also had default values for year, month, and day (they must be in the legal numeric range, and cannot be `None`), so you'd get a `time.struct_time` value that described the date "1970-Jan-01, 23:01:60" which is not what we want.

Reply:
gotcha, interesting

Reply:
The follow-on question is how often are you going to get a 'time' *without* a date that has a seconds value of 60?  Probably almost never, but the ODL spec allows it ...